README.md

```diff
@@ -79,12 +98,114 @@ For example, the incoming message contains the following body:
   }
 ]
 ```
-You have a table called `itemstable` with fields: `id`, `text`, `createdAt`.
-For this purpose you need to specify the following SQL template:
-```$sql
-INSERT INTO itemstable(id, text, createdAt) VALUES(${id}, ${text}, current_timestamp)
+All changes will rollback, if something wrong with data.
+
+## General Sql Query Action
+
+**Expert mode**. You can execute SQL query or SQL script in this action.
+
+### Configuration field
+
+#### SQL Query 
+Put your SQL expression to `SQL Query` for further execution.
+You can put only one SQL query or several queries with delimiter `;`.
+All queries are executed in one transaction. All changes will rollback, if something wrong with one of execution.
+Also if you want to use prepared statements in your query,
+you need define prepared statement variables like this way `sqlVariableName = @MetadataVariableName:type` where:
+1. `sqlVariableName` - variable name in sql expression;
+2. `MetadataVariableName` - variable name in metadata (it can be the same as `sqlVariableName`);
+3. `type` - type of variable , following types are supported:
+   * string (also default type if type is omitted)
+   * number 
+   * boolean 
+   
+For example, for sql expression `SELECT * FROM tableName WHERE column1 = 'text' AND column2 = 15` you need to use following template:
+`SELECT * FROM tableName WHERE column1 = @column1:string AND column2 = @column2:number` and put values into generated metadata.
+
+![image](https://user-images.githubusercontent.com/16806832/53731432-635a2200-3e83-11e9-9a4e-0fc26aeeb001.png)
+
+### Input metadata
+Input metadata is generated from `SQL Query` configuration field if this field contains at list one defined value.
```

Comment:
Misprint: 
Input metadata is generated from `SQL Query` configuration field if this field contains at *least* one defined value.